src/components/UptimeViewer/UptimeViewer.tsx

```diff
@@ -17,33 +19,32 @@ interface NodeUptimeProps {
 
 export function NodeUptime({StartTime, DisconnectTime}: NodeUptimeProps) {
     let uptime: string | undefined;
+    let content: React.ReactNode = null;
 
     if (DisconnectTime) {
         uptime = getDowntimeFromDateFormatted(DisconnectTime);
+        content = (
+            <DefinitionList.Item key={'DisconnectTime'} name={i18n('disconnect-time')}>
+                {formatDateTime(DisconnectTime, {withTimeZone: true})}
+            </DefinitionList.Item>
+        );
     } else if (StartTime) {
         uptime = getUptimeFromDateFormatted(StartTime);
+        content = (
+            <DefinitionList.Item key={'StartTime'} name={i18n('start-time')}>
+                {formatDateTime(StartTime, {withTimeZone: true})}
+            </DefinitionList.Item>
+        );
     }
 
     if (!uptime) {
         return EMPTY_DATA_PLACEHOLDER;
     }
+
     return (
         <CellWithPopover
             placement={['top', 'auto']}
-            content={
-                <DefinitionList responsive>
-                    {StartTime ? (
-                        <DefinitionList.Item key={'StartTime'} name={i18n('start-time')}>
-                            {formatDateTime(StartTime, {withTimeZone: true})}
-                        </DefinitionList.Item>
-                    ) : null}
-                    {DisconnectTime ? (
-                        <DefinitionList.Item key={'DisconnectTime'} name={i18n('disconnect-time')}>
-                            {formatDateTime(DisconnectTime, {withTimeZone: true})}
-                        </DefinitionList.Item>
-                    ) : null}
-                </DefinitionList>
-            }
```

Comment:
Previously I assumed, that if node is disconnected, there will be no `StartTime` in the response. However, `StartTime` may be cached somewhere on backend and we could receive both `StartTime` and `DisconnectTime` for disconnected node, in this case we have both values in the tooltip with different timestamps. After this fix `StartTime` will be in popover only if there is no `DisconnectTime`